app/models/edition.rb

```diff
@@ -56,21 +59,30 @@ class ContentItem < ApplicationRecord
   validates :description, well_formed_content_types: { must_include: "text/html" }
   validates :details, well_formed_content_types: { must_include_one_of: %w(text/html text/govspeak) }
 
-  validates :locale, inclusion: {
-    in: I18n.available_locales.map(&:to_s),
-    message: 'must be a supported locale'
-  }
-
   validate :user_facing_version_must_increase
   validate :draft_cannot_be_behind_live
 
-  validates_with VersionForLocaleValidator
+  validates_with VersionForDocumentValidator
   validates_with BasePathForStateValidator
-  validates_with StateForLocaleValidator
+  validates_with StateForDocumentValidator
   validates_with RoutesAndRedirectsValidator
 
+  # Temporary code until we remove content_id and locale fields
```

Comment:
let's mark any sections of temporary code with a `TODO` comment and create a trello card to get rid of them after a few weeks